apps/hello-ui/package.json

```
@@ -26,16 +26,16 @@
     "safari >= 13"
   ],
   "dependencies": {
-    "@module-federation/rsbuild-plugin": "^0.9.0",
+    "@module-federation/rsbuild-plugin": "^2.5.1",
     "shell-ui": "workspace:*",
     "shared": "workspace:*",
```

Biome (2.4.16)

[error] The key shared was already declared.

(lint/suspicious/noDuplicateObjectKeys)